website/docs/cli/init/index.mdx

```diff
@@ -1,12 +1,10 @@
 ---
-page_title: Initializing Working Directories - Terraform CLI
+page_title: Initialize the Terraform working directory
```

Comment:
We've included the word "overview" in other index pages, do we want the same here?